src/cryptoadvance/specter/templates/includes/coin_selection.html

```diff
@@ -0,0 +1,232 @@
+                        return 0;
+                    }
+                }
+            } else {
+                spendableAmount = this.getCoinSelectedAmount;
+            }
+    
+            return (unit == 'sat' ? spendableAmount * 100000000 : spendableAmount);
+        }
+
+        /**
+         * Updates the lightDOM: It creates one hidden input-element for each tx which is selected in the shadowDOM
+         */
+        updateLd() {
+            while (this.ld.div.lastChild) {
+                this.ld.div.removeChild(this.ld.div.lastChild);
+            }
+            this.txtable.getSelectedTxs().forEach( (tx) => {
+                var element = document.createElement("input");
+                element.type = "hidden";
+                element.name = "coinselect"
+                element.classList.add("coinselect-hidden") // for cypress-tests
+                element.value= tx.txid + "," + tx.vout
+
+                this.ld.div.appendChild(element)
+            })
+        }
+
+        getCoinSelectedAmount() {
+            var sum = 0
+            this.txtable.getSelectedTxs().forEach( (tx) => {
+                sum = sum + tx.amount
+            })
+            return sum
+        }
+    
+        toggleCoinselection() {
+            if (this.coinselectionDiv.style.display === 'block') {
+                this.coinselectionDiv.style.display = 'none';
+                this.toggler.innerHTML = `{{ _("Coinselection") }} &#9654;`;
+                if (this.isCoinSelectionActive()) {
+                    toggleExpand();
+                }
+            } else {
+                this.coinselectionDiv.style.display = 'block';
+                this.toggler.innerHTML = `{{ _("Coinselection") }} &#9660;`;
+            }
+        }
+
+        /*
+        * Removes all coin_selection if it's no longer active
+        */
+        toggleExpand() {
+            if (this.isCoinSelectionActive()) {
+                setVisibility('coinselection', 'none');
+            } else {
+                setVisibility('coinselection', 'block');
+
+                let coins = document.getElementsByClassName('coin_select_checkbox');
+                // unselect all choices
+                for(var i = 0; i < coins.length; i++){
+                    coins[i].checked = false;
+                }
+            }
+            this.fireChangeEvent();
+        }
+
+        /**
+        * Fetches assetBalance from the Specter API 
+        */
+        async fetchAssetBalances(wallet_alias) {
+            let url = `{{ url_for('wallets_endpoint_api.asset_balances', wallet_alias='WALLET_ALIAS') }}`.replace("WALLET_ALIAS", this.walletAlias);
+            var formData = new FormData();
+            formData.append('address', this.address);
+            formData.append('csrf_token', '{{ csrf_token() }}');
+            try {
+                const response = await fetch(
+                    url,
+                    {
+                        method: 'POST',
```

Comment:
Shouldn't this be a GET when just retrieving data? (that would also solve the `POST` issue in `wallets_api.py`).

Reply:
Yes, done!